src/main/java/frc/robot/RobotContainer.java

```diff
@@ -58,16 +146,82 @@ public RobotContainer() {
      */
     private void configureButtonBindings() {
         /* Driver Buttons */
-        zeroGyro.onTrue(new InstantCommand(() -> s_Swerve.zeroHeading()));
+        driverY.onTrue(new InstantCommand(() -> s_Swerve.zeroHeading()));
+        // driverA.whileTrue(new AimShoot(s_Swerve, s_ShooterPivot, s_Shooter));
+        
+        //driverA.whileTrue(new InstantCommand(() -> s_Shooter.setShooterVoltage(6, -6))).onFalse(new InstantCommand(() -> s_Shooter.stop()));
+        //driverX.onTrue(new ShooterFrontRollerRun(s_Shooter));
+        driverB.toggleOnTrue(new TeleopSwerve(
+                s_Swerve, 
+                () -> -driver.getRawAxis(driverLeftY), 
+                () -> -driver.getRawAxis(driverLeftX), 
+                () -> -driver.getRawAxis(driverRightX),
+                () -> driverDpadUp.getAsBoolean(),
+                () -> s_Swerve.getTargetRotation(),
+                () -> driverLeftTrigger.getAsBoolean(),
+                rotationSpeed,
+                true
+            )//.until(() -> Math.abs(s_Swerve.getGyroYaw().getDegrees() % 360) < s_Swerve.getTargetRotation() + Constants.AUTO_ROTATE_DEADBAND && 
+            //Math.abs(s_Swerve.getGyroYaw().getDegrees() % 360) > s_Swerve.getTargetRotation() - Constants.AUTO_ROTATE_DEADBAND)
+        );
+
+        driverX.onTrue(new ParallelCommandGroup(
```

Comment:
add comments describing what the command does as a whole as well as what each individual command does if not obvious